src/backbone.marionette.js

```diff
@@ -0,0 +1,6 @@
+/*
+const Marionette = {
+};
+
+export default Marionette;
+*/
```

Comment:
I think this was overwriting the main object causing failures on test when I first ran this. Thought of just leaving this commented out in the meantime and handled in es6 modules.